app/services/stripe/invoice_handler.rb

```diff
@@ -26,6 +25,7 @@ def update_invoice_paid_timestamp!
   private
 
   def get_brand
-    PaymentHandler.find_by(token: @invoice_data.customer).try(:brand)
+    brand_id = PaymentHandler.where(token: @invoice_data.customer).pluck(:brand_id).first
+    raise StandardError.new('Could not find PaymentHandler for that customer')  unless brand_id
```

Comment:
You probably don't need to raise an error here. If a payment handler isn't found, because we said that `brand_id` can't be null, it'll raise there anyway.